yocto/myencfs_git.bb

```diff
@@ -0,0 +1,46 @@
+SUMMARY = "myencfs example application"
+DESCRIPTION="A simple directory encryption/decryption using authenticated encryption"
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://COPYING;md5=9859f02baa869af387514f9e5dd8cc3e \
+                    file://debian/copyright;md5=358aa3ad2f63e0c87d59d3150ef3fd7e"
+
+SRC_URI = "git://github.com/alonbl/myencfs.git;protocol=https"
+
+SRCREV = "${AUTOREV}"
+S = "${WORKDIR}/git"
+
+# Crypto can be either openssl_crypto or mbedtls_crypto
+MYENCFS_CRYPTO ?= "openssl_crypto"
+# List of enabled features that can be overriden
+PACKAGECONFIG ?= "tools encrypt decrypt bio-file test fuse static-libs ${MYENCFS_CRYPTO}"
+
+
+PACKAGECONFIG[tools]	= "--enable-tool"
```

Comment:
usually it is best practice to provide both enable and disable as you never knows what the default of the package and when it changes. Gentoo is doing that automatically, I see that yocto does not have these helpers.

Reply:
Actually it is defined implicitly. PACKAGECONFIG has six parameters which can be explicitly defined. I used it in the crypto_openssl for example. The second empty field is the disable field which is left empty. When I don't set --enable-tool for example, it is the disablement of tools. I hope I was clear

Reply:
I think you should explicitly disable the feature as if the upstream modifies the defaults, you may end up with unintentionally enabled feature.

what are the 5th and 6th parameters? where can I read?

Reply:
I've provided you with a link in a previous comment

Reply:
Added an explicit disable